tests/flow/tests_onnx.py

```diff
@@ -112,6 +112,7 @@ def test_onnx_modelrun_mnist_autobatch(env):
     env.assertEqual(ret[9], 2)
     env.assertEqual(len(ret[11]), 1)
     env.assertEqual(len(ret[13]), 1)
+    env.assertEqual(ret[15], 0)
```

Comment:
why not test `0` for all `modelget` commands where `minbatchsize` was not set?

Reply:
Added test 'minbatchtimeout' for every time we test 'minbatchsize'